lib/Fhp/Segment/WPD/HKWPDv5.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/** @noinspection PhpUnused */
+
+namespace Fhp\Segment\WPD;
+
+use Fhp\Segment\BaseSegment;
+use Fhp\Segment\Paginateable;
+
+/**
+ * Segment: Depotaufstellung anfordern (Version 5)
+ *
+ * @link https://www.hbci-zka.de/dokumente/spezifikation_deutsch/fintsv3/FinTS_3.0_Messages_Geschaeftsvorfaelle_2015-08-07_final_version.pdf
+ * Section: C.4.3.1a
+ */
+class HKWPDv5 extends BaseSegment implements Paginateable
+{
+    /** @var \Fhp\Segment\Common\KtvV3 */
+    public $depot;
+
+    /** @var string|null */
+    public $waehrungDerDepotaufstellung;
+
+    /** @var \Fhp\Segment\Common\Kursqualitaet|null */
+    public $kursqualitaet;
+
+    /** @var int|null Only allowed if {@link ParameterDepotaufstellungV2::$eingabeAnzahlEintraegeErlaubt} says so. */
+    public $maximaleAnzahlEintraege;
+
+    public static function create(\Fhp\Segment\Common\KtvV3 $ktv): HKWPDv5
+    {
+        $result = HKWPDv5::createEmpty();
+        $result->depot = $ktv;
+        return $result;
+    }
+
+    public function setPaginationToken(string $paginationToken)
+    {
+        $this->aufsetzpunkt = $paginationToken;
```

Comment:
@ZwenAusZwota I was just looking over this code for some cleanups, and I noticed that:

1. The specification linked in line 12 is about version 6, but this here encodes version 5. Which is the right version? Which version(s) does your bank support?
2. The field `aufsetzpunkt` that is set here doesn't actually exist in this class, but it is specified in version 6. Should that field be added?